### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by JosiahParry** [shssoichiro/maybe-rayon#2]
This crate looks quite appealing because I am working on a project that cross compiles to wasm32 targets. Instead of panicking, I'd like for the crate to fallback to a standard iterator. Will this accomplish that?

**Comment by shssoichiro** [comment]
Yes, that is exactly what this crate does. It should require very few updates, because it simply re-exports rayon when rayon is enabled, and uses standard iterators when rayon is disabled.
